spellbook/macros/alter_tblproperties/alter_tblproperties_balances_ethereum_erc20_day.sql

```diff
@@ -0,0 +1,11 @@
+{% macro alter_tblproperties_balances_ethereum_erc20_day() -%}
+{%- if target.name == 'prod'-%}
+ALTER VIEW balances_ethereum.erc20_day SET TBLPROPERTIES('dune.public'='true',
+                                                    'dune.data_explorer.blockchains'='["ethereum"]',
+                                                    'dune.data_explorer.category'='abstraction',
+                                                    'dune.data_explorer.abstraction.type'='sector',
+                                                    'dune.data_explorer.abstraction.name'='balances',
+                                                    'dune.data_explorer.contributors'='["soispoke","dot2dotseurat"]');
+{%- else -%}
+{%- endif -%}
+{%- endmacro %}
```

Comment:
Repeating this for every file seems like it could get unwieldy fast! :sweat_smile:

Reply:
Agreed! We have plans to write a macro that loops across selected models